doc/index.rst

```diff
@@ -126,6 +128,8 @@ licensing, as described in :ref:`Zephyr_Licensing`.
    Indices and Tables
    ******************
```

Comment:
should likely be excluded, too

Reply:
now we have an index for tex and removing lots of :only: usage in main index